test/JDBC/expected/babel_operators.out

```diff
@@ -3,12 +3,12 @@ SELECT (123 + 0x42);
 SELECT (0x42 + 123);
 GO
 ~~START~~
-bigint
+int
```

Comment:
Make sure that these all changes are expected T-SQL behaviour

Reply:
T-SQL does choose int as the result type here. Note that as a result of using INT instead of BIGINT for the result type, we can see integer overflows that didn't previously occur in babelfish. But, these overflows match T-SQL behavior.